### PR TITLE
Feature/main cleanup

### PR DESCRIPTION
**Pull request by simpleman19**
This cleans up some TODOs and fixes delivery instructions stub-out

### CODE REVIEW

File: src/acme/pd/Path.java [hunk 1]
```diff
@@ -45,10 +48,28 @@ public void setBlocksBetweenPickupandDropoff(int blocksBetweenPickupandDropoff)
    
 
     public ArrayList<String> getDeliveryInstructions() {
-        // TODO implement delivery instructions
         ArrayList<String> instructions = new ArrayList<>();
-        Random rand = new Random();
-        IntStream.range(0, 10).forEach(i -> instructions.add("Go " + Direction.values()[rand.nextInt() % 4] + " 1 block"));
+        if (this.path != null && this.path.size() > 0) {
+            for (int i = 0; i < this.path.size() - 1; i++) {
+                instructions.add("Go from " + this.path.get(i).getIntersectionName() + " to "
```
Comment (jake6642) [inline src/acme/pd/Path.java]: DIdn't Harbert say specifically that he wanted E/W/N/S directions in the instructions? Maybe this should say something like "Go (direction) on (roadName) to (IntersectionName)"

Comment (simpleman19) [reply src/acme/pd/Path.java]: I guess I can do that

File: src/acme/pd/Map.java [hunk 2]
```diff
@@ -319,6 +318,48 @@ private MapIntersection findIntersection(String nsRoad, String ewRoad) {
         return null;
     }
 
+    public Direction getTravelDirection(MapIntersection start, MapIntersection end) {
```
[the remark this answers is not in the document]
Comment (jake6642) [reply src/acme/pd/Map.java]: If this wasn't one of the main purposes of the system, I wouldn't care

[the remark this answers is not in the document]
Comment (simpleman19) [reply src/acme/pd/Map.java]: Fine...

[the remark this answers is not in the document]
Comment (simpleman19) [reply src/acme/pd/Map.java]: I'm honestly just trying to get this system working at this point

**Comment by simpleman19** [comment]
Check that

**Comment by epielemeier** [comment]
Here is the current output (for a pickup customer at 3&D and a dropoff customer at 3&B):
image

### Correctness
This is displaying the correct path.

### Content
The [design doc](https://docs.google.com/document/d/1KWQUDQkyZNIrBAvCBPbOJA1iciarothrF2RA4RoMk20/edit#heading=h.2u6wntf) specified how this was supposed to look:

image

The current output is missing the **number of blocks the courier travels at each step**.

It sucks to re-do things, but it also sucks for reviewers when you ignore the spec we worked so hard on.

Since our code freeze is scheduled for tomorrow I suggest we go ahead and merge this in as-is.

**Comment by jake6642** [comment]
Somehow, reseeding did the trick. I agree with Emily, and technically print delivery path is @pnoorossana 's story, but I don't think she is going to have time to finish it. If she does, she has something to work off of now. I'm merging it in.

**Comment by epielemeier** [comment]
I just noticed another missing piece in this:
image

We should also be displaying "Pick up package at XXX Intersection" / "Drop off package at XXX Intersection"

**Pull request merged**